WcaOnRails/app/webpacker/javascript/wca-live/attempts.js

```diff
@@ -25,11 +28,19 @@ function centisecondsToClockFormat(centiseconds) {
  */
 function decodeMbldAttemptResult(value) {
   if (value <= 0) return { solved: 0, attempted: 0, centiseconds: value };
-  const missed = value % 100;
-  const seconds = Math.floor(value / 100) % 1e5;
-  const points = 99 - (Math.floor(value / 1e7) % 100);
-  const solved = points + missed;
-  const attempted = solved + missed;
+  if (value < 1e9) { // New-style result.
```

Comment:
This if-check hides the actual design decision behind the format, that is "results starts with a 1".

Even though your approach might be computationally efficient, I would rather write something along the lines of `value.toString().startsWith('1')` for the sake of clarity/readability.

Reply:
Additional nitpick: I would have the if-query check whether or not it is an _old_ style result, and treat the new-style result as the "default assumption" below the if.

On that note, maybe extract the if-condition into its own `isOldStyleResult` variable and then use it through `if (isOldStyleResult)`

Reply:
Done.  For the first suggestion, I padded it to a 10-digit number first (otherwise if someone gets 80+ points this logic treat it as an old-style result).